src/channeldef.ts

```diff
@@ -472,14 +475,12 @@ export interface PositionBaseMixins {
 
 export interface BandMixins {
   /**
-   * For rect-based marks (`rect`, `bar`, and `image`), mark size relative to bandwidth of [band scales](https://vega.github.io/vega-lite/docs/scale.html#band), bins or time units. If set to `1`, the mark size is set to the bandwidth, the bin interval, or the time unit interval. If set to `0.5`, the mark size is half of the bandwidth or the time unit interval.
-   *
-   * For other marks, relative position on a band of a stacked, binned, time unit or band scale. If set to `0`, the marks will be positioned at the beginning of the band. If set to `0.5`, the marks will be positioned in the middle of the band.
+   * Relative position on a band of a stacked, binned, time unit or band scale. If set to `0`, the marks will be positioned at the beginning of the band. If set to `0.5`, the marks will be positioned in the middle of the band.
```

Comment:
```suggestion
   * Relative position on a band of a stacked, binned, time unit, or band scale. If set to `0`, the marks will be positioned at the beginning of the band. If set to `0.5`, the marks will be positioned in the middle of the band, etc.
```

Reply:
I don't think we need etc at the end.

Reply:
Maybe say "and so on." then or start with "For example..."? When I read it I was surprised to only see two examples but the text makes it sound like this is all you can set.